curriculum/challenges/english/20-upcoming-python/learn-regular-expressions-by-building-a-password-generator/6564d97a5ef99783216a6229.md

```diff
@@ -0,0 +1,64 @@
+---
+id: 6564d97a5ef99783216a6229
+title: Step 44
+challengeType: 20
+dashedName: step-44
+---
+
+# --description--
+
+Python provides a particular type of strings called *raw* strings. Raw strings are prefixed with a `r` and the main difference respect to a regular string is that they treat the backslash character as a regular character and not as an escape character. When writing regular expressions, using raw strings is a good practice, since they can usually contain a lot of `\` characters.
```

Comment:
```suggestion
Python provides a particular type of string called *raw* strings. Raw strings are prefixed with a `r`. The key distinction from regular strings lies in how they handle the backslash character: in raw strings, backslashes are treated as literal characters rather than escape characters. When writing regular expressions, using raw strings is a good practice, since they can usually contain a lot of `\` characters.
```